lib/improver/cli/__init__.py

```diff
@@ -220,6 +220,36 @@ def with_intermediate_output(wrapped, *args, intermediate_output=None,
     return result
 
 
+def create_constrained_inputcube_converter(*constraints):
```

Comment:
Also can we move this whole block of code next to the other converters, i.e: above the comment line `# output handling`

Reply:
Done